portal/src/api/api.ts

```diff
@@ -1009,6 +1009,33 @@ class FKApi {
             url: this.baseUrl + "/status",
         });
     }
+
```

Comment:
Can you please add typescript return types and parameter types? Trying to get these in when I can.